java-frontend/src/main/java/org/sonar/java/se/xproc/BehaviorCache.java

```diff
@@ -51,7 +52,7 @@ public MethodBehavior methodBehaviorForSymbol(Symbol.MethodSymbol symbol) {
   @CheckForNull
   public MethodBehavior get(Symbol.MethodSymbol symbol) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 16 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3776)